tests/model/test_estimation.py

```diff
@@ -189,13 +193,36 @@ def test_getitem():
 
 
 def test_properties():
-    a = EstimationStep.create('foce', epsilon_derivatives=['EPS(1)'])
-    assert a.epsilon_derivatives == ('EPS(1)',)
-    a = EstimationStep.create('foce', eta_derivatives=['ETA(1)'])
-    assert a.eta_derivatives == ('ETA(1)',)
-    a = EstimationStep.create('foce', predictions=['PRED'])
+    d = tuple(map(Expr.symbol, ('EPS(1)',)))
```

Comment:
Pharmpy symbols should not contain parentheses. (parsing becomes ambiguous)

Reply:
Switched the symbols to instead use underlines ("ETA_1")